styles.css

```diff
@@ -0,0 +1,110 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap');
+
+/* resets */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --hover-btn: #6070ff;
+  --active-btn: #2230d2;
+  --disabled-text: #5e6c84;
+  --enabled-text: #6070ff;
+  --secondary-color: #ffff;
+  --primary-color: #396df2;
+  --text-color: #091e42;
+  --font-fam: 'Poppins', sans-serif;
+  --fade-bg: #ebebff;
+  --icon-color: #505f79;
+  --semi-bold: 700;
+  --text-color2: #7f8cff;
+  --main-bg: #e5e5e5;
+}
+
+body {
+  background-color: var(--main-bg);
+}
+
+a {
+  text-decoration: none;
+}
+
+/* end of resets */
+
+.nav-menu a {
+  color: var(--primary-color);
+  font-family: var(--font-fam);
+  font-weight: 700;
+  font-size: 1.5 rem;
+  line-height: 20px;
+}
+
+/* header section */
+
+header {
+  background-color: var(--secondary-color);
+  border: 3px solid var(--secondary-color);
+}
+
+.nav-menu {
+  display: flex;
+  margin: 14px 24px 14px 24px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+/* hero section */
+
+#hero {
+  background-image: url(images/header-bg.png);
+  height: 42rem;
+  background-repeat: no-repeat;
+  position: absolute;
+  background-color: var(--secondary-color);
+  border-bottom-left-radius: 100px;
+  display: flex;
+}
+
+.headlines {
+  margin: 12px 24px 0 24px;
+}
+
+.headlines h1 {
+  font-family: var(--font-fam);
+  color: var(--text-color);
+  font-weight: var(--semi-bold);
+  line-height: 3.25rem;
+  font-size: 3rem;
+}
+
+.headline-title h1 {
+  margin: 114px 0 0 0;
```

Comment:
I kindly suggest you to use flexbox to center elements in the container, this practice is good to adjust to any device size

Reply:
I don't know how to do it, can you help me? The project has been returned to me many times, and I have already corrected everything, I feel frustrated because now you ask me for something new.